### PR TITLE
Kdesktop 1251 Some sentry are not readable

### PR DESCRIPTION
**Pull request by herve-er**

### CODE REVIEW

File: src/libcommon/log/customlogwstream.h [hunk 1]
```diff
@@ -26,19 +26,43 @@ class CustomLogWStream : private std::wstringstream {
...
+            std::wstringstream::operator<<(ul);
+            return *this;
+        }
+        CustomLogWStream &operator<<(double d) {
+            std::wstringstream::operator<<(d);
+            return *this;
+        }
         CustomLogWStream &operator<<(const std::wstring &str) {
-            std::wstringstream::operator<<(str.c_str());
+            CustomLogWStream::operator<<(str.c_str());
```
Comment (luc-guyot-infomaniak) [inline src/libcommon/log/customlogwstream.h:65]: Is it useful or required to call `c_str()`?

Comment (herve-er) [reply src/libcommon/log/customlogwstream.h:65]: As for CustomLogWStream &operator<<(const wchar_t *str), the operator is defined outside of std::wstringstream. Here is a commit where I used the same workaround for both CustomLogWStream &operator<<(const wchar_t *str) and CustomLogWStream &operator<<(const std::wstring &str).

Comment (herve-er) [reply src/libcommon/log/customlogwstream.h]: [893c0da](https://github.com/Infomaniak/desktop-kDrive/pull/321/commits/893c0da73b65eade834cfb5c0fb4eba7b42ab79f)

File: src/libcommon/log/customlogwstream.h [hunk 2]
```diff
@@ -26,19 +26,45 @@ class CustomLogWStream : private std::wstringstream {
...
-            // inheritance
             static_cast<std::wstringstream &>(*this) << str;
             return *this;
         }
+        CustomLogWStream &operator<<(const std::wstring &str) {
+            static_cast<std::wstringstream &>(*this) << str;
+            return *this;
+        }
+
+        CustomLogWStream &operator<<(bool b) {
```
Comment (luc-guyot-infomaniak) [inline src/libcommon/log/customlogwstream.h:42]: Maybe for later: using `std::boolalpha` would print `true` and `false` instead of `1` and `0`.

**Pull request merged**